tests/unit/schema.test.js

```diff
@@ -124,7 +124,7 @@ const exampleCredential = {
    ],
    "issuanceDate": "2020-03-18T19:23:24Z",
    "credentialSchema": {    // this is the schema
-      "id": "schema:dock:5C78GCA",
+      "id": "blobmodule:dock:5C78GCA",
```

Comment:
is this correct? `blobmodule` not `blob`?

Reply:
It should be a `blob`